tests/parsely/test-mu-parsely-integration.php

```diff
@@ -6,10 +6,8 @@
 use Parsely\UI\Row_Actions;
 use WP_UnitTestCase;
 
-function test_mode() {
-	$mode = getenv( 'WPVIP_PARSELY_INTEGRATION_TEST_MODE' );
-	return $mode ?: 'disabled';
-}
```

Comment:
Created a common `utils` folder and add common functions there which we are using more than once and across multiple tests.